# A message representing all data from a single NMEA0183 sentence
Header header

float64 wind_angle
string reference
float64 wind_speed
string speed_unit
string status

